### PR TITLE
feat: allow symfony 7 (WIP)

### PR DESCRIPTION
**Pull request by Chris53897**
Just a quick check for symfony 7 support.
there are dependencies that do not have symfony 7 support yet.

### CODE REVIEW

File: tests/Unit/Logs/TestHook/DeprecationTest.php [hunk 1]
```diff
@@ -33,6 +33,7 @@ protected function createEvent(): DeprecationTriggered
             123,
             false,
             false,
+            false,
```
Comment (Jean85) [inline tests/Unit/Logs/TestHook/DeprecationTest.php]: Why did you add this?

**Review by Jean85 — COMMENTED** [review]
It should work immediately, since we do not have any active deprecation. You broke the build though...

File: tests/Unit/Logs/TestHook/DeprecationTest.php (hunk 1 above)
Comment (Chris53897) [reply tests/Unit/Logs/TestHook/DeprecationTest.php]: I have testet it locally by adding these line to composer.json `,
    "minimum-stability": "dev",
    "prefer-stable": false`

With that i got more dev-packages that i wanted.
And there is a new parameter for that function call in stability `dev` that broke testing locally.

Comment (Chris53897) [reply tests/Unit/Logs/TestHook/DeprecationTest.php]: I did revert the change. But it will be needed in the future i guess.

**Comment by Chris53897** [comment]
CI is green. But there is no testrun for symfony 7.
Maybe wait for 1 or 2 weeks, until there is the first stable release of symfony 7?

**Comment by keradus** [comment]
there is stable Sf7 now

**Comment by Chris53897** [comment]
I did rerun CI.
Symfony/Console 7 is blocked by https://github.com/facile-it/facile-coding-standard/pull/63

@Jean85 Can you please have a look?

**Review by Jean85 — APPROVED** [review]
